solenoid/elements/xml_handlers.py

```diff
@@ -95,6 +101,11 @@ def parse_author_pubs_xml(xml_gen, author_data):
             # Paper does not have a library status
             if entry.find(".//api:library-status", NS):
                 continue
+            # Publication type is either a journal article, book chapter, or
+            # conference proceeding
+            pub_type = extract_attribute(entry, ".//api:object", "type-id")
+            if not (pub_type == '5' or pub_type == '3' or pub_type == '4'):
```

Comment:
A more pythonic way to write this would be:
```python
if pub_type not in ('3', '4', '5'):
```

Reply:
Ha, yes indeed. Thank you!